test/cluster/test_strong_consistency.py

```
@@ -1760,3 +1760,832 @@ async def _run(ops, *, kind):
                 await run_counter_batch([
                     ("counter_add", (1, 1)),
                 ], kind="counter")
+
+async def test_tablet_migration(manager: ManagerClient):
```

Ruff (0.16.3)

[warning] Too many statements (53 > 50)

(PLR0915)